src/components/themed/SideMenu.tsx

```diff
@@ -342,23 +346,23 @@ export function SideMenu(props: DrawerContentComponentProps) {
 
   const navigationRows = (
     <>
-    <View style={styles.rowsContainer}>
-      <ScrollView overScrollMode="always" scrollIndicatorInsets={SCROLL_INDICATOR_INSET_FIX}>
-        {rowDatas.map(rowData => (
-          <EdgeTouchableOpacity accessible={false} onPress={rowData.pressHandler} key={rowData.title} style={styles.rowContainer}>
-            <View style={styles.rowIconContainer}>
-              {rowData.iconName != null ? <Fontello name={rowData.iconName} style={styles.icon} size={theme.rem(1.5)} color={theme.iconTappable} /> : null}
-              {rowData.iconNameFontAwesome != null ? (
-                <FontAwesome5Icon name={rowData.iconNameFontAwesome} style={styles.icon} size={theme.rem(1.25)} color={theme.iconTappable} />
-              ) : null}
-            </View>
-            <View style={styles.rowBodyContainer}>
-              <TitleText style={styles.text}>{rowData.title}</TitleText>
-            </View>
-          </EdgeTouchableOpacity>
-        ))}
-      </ScrollView>
-    </View>
+      <View style={styles.rowsContainer}>
```

Comment:
It seems like the previous commit must have contained a whitespace error. Could you double-check the linter on "Split up giant JSX in the SideMenu render"?

Reply:
Yeah probably from staging partial diffs to make the commits more atomic, will check